reference/forms/types/entity.rst

```diff
@@ -110,11 +110,12 @@ choice_label
     The ``choice_label`` option was introduced in Symfony 2.7. Prior to Symfony
     2.7, it was called ``property`` (which has the same functionality).
 
-**type**: ``string``
+**type**: ``string`` or a Closure
```

Comment:
actually it should be `null|callable|string|PropertyPath` see https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php#L162